pkgs/applications/misc/wtf/default.nix

```diff
@@ -9,28 +9,23 @@ buildGoModule rec {
   pname = "wtf";
   version = "0.21.0";
 
+  overrideModAttrs = _oldAttrs : _oldAttrs // {
+    preBuild = ''export GOPROXY="https://gocenter.io"'';
```

Comment:
Should we have this for all packages?

Reply:
I am not sure, but I am also a bit out of my element here; not really using Go and not knowing its best practices.

At least, this is what the wtf project's build instructions advice. And the "immutable re-usable Go modules" part sounds quite interesting. But, I also here that with newer Go versions, there will be an official proxy / modules server... ?!